docs/main/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/rs-configuration.md

```diff
@@ -69,6 +69,11 @@ EOF
 ```
 
 The diskpool is created with labels to satisfy `poolHasTopologyKey` and `poolAffinityTopologyLabel` parameters of the storage class. This helps place the replicas of volume based on the pool labels.
+You can use the plugin to add labels to pools that do not have topology labels:
```

Comment:
The content seems fine, but the intent is missing. Why / when does a user need to do this?

Reply:
Inorder to place the replicas of a volume based on the pool labels i.e. to satisfy `poolHasTopologyKey` and `poolAffinityTopologyLabel` parameters of the storage class , the pools must be labelled with the topology field.  
This can be achieved in two ways.
1. The existing pools can be labelled using the plugin : (Note : There will not be any effect on all the pre-existing volumes)
```
kubectl mayastor label pool pool-on-node-1 topology-key=topology-value -n openebs
```
2. Create a new pool with the labels

**Example DiskPool Definition with Labels**
```
cat <<EOF | kubectl create -f -
apiVersion: "openebs.io/v1beta2"
kind: DiskPool
metadata:
  name: pool-on-node-1
  namespace: openebs
spec:
  node: workernode-1-hostname
  disks: ["/dev/disk/by-id/<id>"]
  topology:
    labelled:
      topology-key: topology-value
EOF
```

Reply:
@avishnu  Does this seems ok ?

Reply:
Seems ok, change the order of (1) and (2).

Reply:
The suggested changes are done :)